### PR TITLE
feat: update default hero

### PR DESCRIPTION
**Pull request by paulpopus**
Old:
image

image

New:
image
image

### CODE REVIEW

**Comment by PatrikKozak** [comment]
@paulpopus The spacing between the `breadcrumbs` and the `heading` seems too large. I'd suggested removing the `top` and `bottom` `margin` on the breadcrumb itself - by targeting it through the default hero though

Feel free to adjust the spacing differently if you think there is a better way

Screen Shot 2024-01-30 at 11 26 04 AM

File: src/components/Hero/Default/index.module.scss [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
+  margin-bottom: calc(var(--base) * 2);
+
+  @include small-break {
+    margin-bottom: calc(var(--base) * 0.8);
+  }
 }
 
 .richText {
   @include small-break {
     margin-bottom: var(--base);
```
Comment (PatrikKozak) [inline src/components/Hero/Default/index.module.scss:24]: @paulpopus The `row gap` coming from `.container` already gives enough spacing between the heading and the bottom paragraph in mobile. I would probably remove this `margin-bottom` here

File: src/components/Hero/Default/index.tsx [hunk 2]
```diff
@@ -10,39 +11,32 @@ import { Page } from '@root/payload-types'
 import classes from './index.module.scss'
 
 export const DefaultHero: React.FC<
-  Pick<Page['hero'], 'richText' | 'sidebarContent'> & {
+  Pick<Page['hero'], 'richText' | 'description'> & {
     breadcrumbs?: Page['breadcrumbs']
   }
-> = ({ richText, sidebarContent, breadcrumbs }) => {
+> = ({ richText, description, breadcrumbs }) => {
   const withoutSidebar =
```
Comment (PatrikKozak) [inline src/components/Hero/Default/index.tsx:18]: @paulpopus Rename this to `withoutDescription`

**Comment by PatrikKozak** [comment]
@paulpopus - is this white box in the design meant to be media for this default hero or no? What is it?

Screen Shot 2024-01-30 at 11 34 07 AM

**Comment by paulpopus** [comment]
> @paulpopus - is this white box in the design meant to be media for this default hero or no? What is it?
> 

Just placeholder

**Pull request merged**